.github/dependabot.yml

```diff
@@ -9,18 +9,32 @@ updates:
       interval: "daily"
     groups:
       azure:
-        "github.com/Azure/*"
+        patterns:
+          - "github.com/Azure/*"
       aws:
-        "github.com/aws/*"
+        patterns:
+          - "github.com/aws/*"
       google:
-        "cloud.google.com/*"
+        patterns:
+          - "cloud.google.com/*"
       elastic:
-        "github.com/elastic/*"
+        patterns:
+          - "github.com/elastic/*"
       k8s:
-        "k8s.io/*"
+        patterns:
+          - "k8s.io/*"
+    labels:
+      - "backport-skip"
+      - "dependency"
+      - "go"
+    open-pull-requests-limit: 10
```

Comment:
seems a bit much

Reply:
It should be a one-off and we probably don't want to miss on updates because of PR limits. Let's see how it plays out.